maintainer/deploy_docs.sh

```diff
@@ -44,12 +44,12 @@ cd ${MDA_DOCDIR} || die "Failed to 'cd ${MDA_DOCDIR}'. Run from the top level of
 mkdir ../${VERSION} && mv * ../${VERSION}
 
 git init
-git config user.name "${GIT_CI_USER}"
-git config user.email "${GIT_CI_EMAIL}"
+git config user.name github-actions
+git config user.email github-actions@github.com
 
 mv ../${VERSION} $VERSION
 
-git remote add upstream "https://${GH_TOKEN}@${GH_REPOSITORY}"
+git remote add upstream "https://github-actions:${GH_TOKEN}@${GH_REPOSITORY}"
```

Comment:
A user name is needed now? 

Does it in any way relate to the `user.name` config?

If this is the case then I would make the script more robust by also passing the name (as env var or commandline argument); I would avoid having hard-coded dependencies between the gh-actions script and the deploy script.